### PR TITLE
fix: remove duplicate import and no relative import

### LINKED ISSUE #3940: Remove duplicated imports and Apply no-relative-imports
There have been a bunch of warnings while running the app due to the following.
References :
https://eslint.org/docs/rules/no-duplicate-imports
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-relative-parent-imports.md

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3940 
fixing es-lint warning for duplicate import and relative parent imports

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4956970&bid=16380975)

**Comment by kushthedude** [comment]
Isnt @snitin315 working on it ?

**Comment by snitin315** [comment]
@maze-runnar you should have notified before making PR as per the guidelines. Its ok for this issue . Hope you won't repeat this again in future.

**Comment by maze-runnar** [comment]
> @maze-runnar you should have notified before making PR as per the guidelines. Its ok for this issue . Hope you won't repeat this again in future.

Sorry , i  just missed that for the first and last time, i will definitily take care of this in future.

**Comment by iamareebjamal** [comment]
Status?

**Comment by maze-runnar** [comment]
> Status?

completed...i think :neutral_face: . is there anything remaining?

File: tests/test-helper.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import Application from '../app';
+import Application from '@ember/application';
 import QUnit from 'qunit';
-import config from '../config/environment';
+import config from 'open-event-frontend/config/environment';
```
Comment (snitin315) [inline tests/test-helper.js]: Are you sure this is correct way?

**Comment by kushthedude** [comment]
Why I see failing checks if the PR is complete ?

**Comment by iamareebjamal** [comment]
> completed...i think neutral_face . is there anything remaining?

Considering that the build is failing, yes

**Comment by kushthedude** [comment]
Does this even work on your local? Is site running after this changes ?

**Comment by maze-runnar** [comment]
> Does this even work on your local? Is site running after this changes ?

yes, it's running with same code.

File: tests/test-helper.js (hunk 1 above)
Comment (maze-runnar) [reply tests/test-helper.js]: @snitin315 , can you please pull these changes to test. it's working as far i tested for everyone's concern if you once please check it.

Comment (maze-runnar) [reply tests/test-helper.js]: > Are you sure this is correct way?

I think it is, other files of project used this type of import.

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4956970&bid=16380975)

File: tests/test-helper.js (hunk 1 above)
Comment (snitin315) [reply tests/test-helper.js]: I'm not sure , but I think somewhere I read that. In ember.js the `project-name/` refers to the  `app` folder by default. Please check if the statement is correct .

Comment (maze-runnar) [reply tests/test-helper.js]: @snitin315 app.js uses same kind of import - 

```
import Application from '@ember/application';
import Resolver from './resolver';
import loadInitializers from 'ember-load-initializers';
import config from 'open-event-frontend/config/environment';

const App = Application.extend({
  modulePrefix    : config.modulePrefix,
  podModulePrefix : config.podModulePrefix,
  Resolver
});

loadInitializers(App, config.modulePrefix);

export default App;

```

**Comment by iamareebjamal** [comment]
The only issue is in the test loader

**Comment by maze-runnar** [comment]
> The only issue is in the test loader

How to resolve this?

**Comment by iamareebjamal** [comment]
By fixing it

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4956970&bid=16380975)

File: tests/test-helper.js (hunk 1 above)
Comment (snitin315) [reply tests/test-helper.js]: [Check This](https://stackoverflow.com/questions/34395590/emberjs-how-to-import-files-using-root-path)

Screenshot from 2020-02-15 17-03-24

Comment (maze-runnar) [reply tests/test-helper.js]: > [Check This](https://stackoverflow.com/questions/34395590/emberjs-how-to-import-files-using-root-path)
> 
> Screenshot from 2020-02-15 17-03-24

>import Application from '@ember/application';
import Resolver from './resolver';
import loadInitializers from 'ember-load-initializers';
import config from 'open-event-frontend/config/environment';
const App = Application.extend({
  modulePrefix    : config.modulePrefix,
  podModulePrefix : config.podModulePrefix,
  Resolver
});
loadInitializers(App, config.modulePrefix);
export default App;

```

when life is unfair :question:

**Comment by maze-runnar** [comment]
> By fixing it

@iamareebjamal , Done!

**Comment by kushthedude** [comment]
Why there another app.js was added ?

**Comment by maze-runnar** [comment]
> Why there another app.js was added ?

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-relative-parent-imports.md
first approach from the given link.

**Comment by iamareebjamal** [comment]
Didn't see that. Reverting

**Pull request merged**